apps/server/src/models/users.ts

```diff
@@ -74,6 +77,19 @@ Users.init(
     otp_expiry: {
       type: DataTypes.DATE,
     },
+    resetToken: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    resetTokenExpiry: {
```

Comment:
what's otp_expiry ?

Reply:
Thats expiry is  for the email verification otp